scripts/build-ovs-for-aarch64-3.3.sh

```diff
@@ -0,0 +1,45 @@
+#!/usr/bin/env bash
+
+VERSION=v3.3.1
+
+# https://docs.openvswitch.org/en/latest/intro/install/debian/#building-open-vswitch-debian-packages
+
+sudo apt-get update
+sudo apt-get install -y \
+    autoconf \
+    automake \
+    build-essential \
+    bzip2 \
+    debhelper-compat \
+    dh-exec \
+    dh-python \
+    dh-sequence-python3 \
+    dh-sequence-sphinxdoc \
+    fakeroot \
+    gcc-aarch64-linux-gnu \
+    graphviz \
+    iproute2 \
+    libcap-ng-dev \
+    libdbus-1-dev \
+    libnuma-dev \
+    libpcap-dev \
+    libssl-dev \
+    libtool \
+    libunbound-dev \
+    openssl \
+    pkg-config \
+    procps \
+    python3-all-dev \
+    python3-setuptools \
+    python3-sortedcontainers \
+    python3-sphinx
+git clone https://github.com/openvswitch/ovs.git
+
+pushd ovs
+git checkout $VERSION
+./boot.sh
+./configure --target=aarch64-buildroot-linux-gnu
```

Comment:
I think this is missing some parameters, I guess `--host=aarch64-buildroot-linux-gnu --build=x86_64-pc-linux-gnu` are needed, maybe also `--prefix=/usr --exec-prefix=/usr --sysconfdir=/etc --localstatedir=/var --program-prefix=` (not sure about those tbh), and finally `--disable-static --enable-shared` which would result in:

```suggestion
./configure --target=aarch64-buildroot-linux-gnu --host=aarch64-buildroot-linux-gnu --build=x86_64-pc-linux-gnu --prefix=/usr --exec-prefix=/usr --sysconfdir=/etc --localstatedir=/var --program-prefix= --disable-static --enable-shared
```

see e.g. this old posting about cross compiling ovs:
https://mail.openvswitch.org/pipermail/ovs-discuss/2017-September/045330.html
and this docs page about autoconf:
https://www.gnu.org/software/autoconf/manual/autoconf-2.69/html_node/Hosts-and-Cross_002dCompilation.html

specifically:

> Now, configure enters cross-compilation mode if and only if --host is passed.